content/docs/1.4.0/snapshots-and-backups/backup-and-restore/restore-recurring-jobs-from-a-backup.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Restore Volume Recurring Jobs from a Backup
+weight: 5
+---
+
+Longhorn can easily restore volume recurring jobs from a backup volume by restoring a backup since v1.4.0. Longhorn can automatically backup volume recurring jobs to the backup volume configuration on the remote backup target since v1.4.0
+
+For more information on how backups work, refer to the [concepts](../../../concepts/#3-backups-and-secondary-storage) section.
+
+When you restore volume recurring jobs from a backup volume by restoring a backup with the setting `Restore Volume Recurring Jobs` enabled, it creates a volume and recurring jobs. If a volume name already exists, the backup and volume recurring jobs will not be restored. If some recurring jobs name already exist, it depends on recurring job settings to reuse the recurring job or create a new recurring job with a specific name.
+
+By default, Longhorn will not automatically restore volume recurring jobs from a backup volume when restoring a backup. You can enable the automatic restoration by Longhorn UI or command line and restore volume recurring jobs from a backup volume by restoring a backup,
+
```

Comment:
```suggestion
For more information on how volume backup works, refer to the [concepts](../../../concepts/#3-backups-and-secondary-storage) section.

When restoring a volume with recurring jobs, Longhorn will restore them together. If the volume name already exists, the volume and the recurring jobs will not be restored.  If the recurring job name already exists but the spec is different, the restoring recurring job will be created with a randomly generated name to avoid conflict. Otherwise, Longhorn will try to reuse existing recurring jobs instead if they are the same as restoring recurring jobs of a backup volume.

By default, Longhorn will not automatically restore volume recurring jobs, users can enable the automatic restoration by Longhorn UI or kubectl.

```

Reply:
Modified. Thanks.